src/music/win32_m.cpp

```diff
@@ -81,15 +81,14 @@ static void TransmitSysex(const uint8_t *&msg_start, size_t &remaining)
 	msg_end++; /* also include sysex end byte */
 
 	/* prepare header */
-	MIDIHDR *hdr = CallocT<MIDIHDR>(1);
-	hdr->lpData = reinterpret_cast<LPSTR>(const_cast<uint8_t *>(msg_start));
-	hdr->dwBufferLength = msg_end - msg_start;
-	if (midiOutPrepareHeader(_midi.midi_out, hdr, sizeof(*hdr)) == MMSYSERR_NOERROR) {
+	auto hdr = std::make_unique<MIDIHDR>(MIDIHDR{
+		.lpData = reinterpret_cast<LPSTR>(const_cast<uint8_t *>(msg_start)),
+		.dwBufferLength = static_cast<DWORD>(msg_end - msg_start)
+	});
```

Comment:
This feels like we're creating a MIDIHDR on the stack, and then made a copy of it in the unique_ptr.